sdk/spring/spring-cloud-azure-starter-appconfiguration-config/README.md

```diff
@@ -168,6 +170,71 @@ spring:
               label-filter: ',${spring.profiles.active}'
 ```
 
+#### Snapshots
+
+App Configuration snapshots allow you to freeze a moment in time of your configuration store. Snapshots are immutable. Snapshots are stored in the same configuration store as the rest of your configuration data. Snapshots are identified by a unique snapshot name. The snapshot name is a string that can contain any combination of alphanumeric characters, hyphens, and underscores. The snapshot name is case sensitive and must be unique within the configuration store.
+
+To load configuration from a snapshot, use the following configuration:
+
+```yaml
+spring:
+  cloud:
+    azure:
+      appconfiguration:
+        stores:
+         -
+           connection-string: <connection-string>
+           selects:
+             -
+              snapshot-name: <snapshot-name>
+```
+
+NOTE: Snapshots have to be of the composition type KEY in order to be loaded, this is to stop configuration name conflicts inside of a snapshot.
+
+When using snapshots, key-filters and label filters aren't used. The snapshot is loaded as is. You can load multiple snapshots by adding multiple selects, even adding key and label filters to other selects.
+
+```yaml
+spring:
+  cloud:
+    azure:
+      appconfiguration:
+        stores:
+         -
+           connection-string: <connection-string>
+           selects:
+             -
+              snapshot-name: <snapshot-name>
+             -
+              key-filter: <key-filter>
+              label-filter: <label-filter>
+```
+
+In this case, the snapshot is loaded first then keys from the filter are loaded. If there are duplicate keys, the last key loaded has the highest priority.
```

Comment:
How can I know which key is loaded lastly? Does the order of loading keys depend on the order of `selects`?

Reply:
Yes it depends on the order in `selects` items closer to the bottom of the list have higher priority than those above them.